tests/ui/async-await/drop-track-bad-field-in-fru.stderr

```diff
@@ -3,6 +3,8 @@ error[E0559]: variant `Option<_>::None` has no field named `value`
    |
 LL |     None { value: (), ..Default::default() }.await;
    |            ^^^^^ `Option<_>::None` does not have this field
+   |
+   = note: all struct fields are already assigned
```

Comment:
perhaps this could be a span-suggestion to remove the field altogether, but honestly I'm fine with just letting the programmer decide what to do with this

Reply:
I think making a suggestion would be better, but it's fine as-is